scan/src/pages/ValidatorHomePage.re

```diff
@@ -259,6 +249,167 @@ let addUptimeOnValidators =
   });
 };
 
+type sort_by_t =
+  | NameAsc // a-z
+  | NameDesc // z-a
+  | VotingPowerAsc // low->high
+  | VotingPowerDesc // high->low
+  | CommissionAsc // low->high
+  | CommissionDesc // high->low
+  | UptimeAsc // low->high
+  | UptimeDesc; // high->low;
+
+let sorting = (validators: array(ValidatorSub.t), sortedBy) => {
+  validators
+  ->Belt.List.fromArray
+  ->Belt.List.sort((a, b) => {
+      switch (sortedBy) {
+      | NameAsc =>
```

Comment:
Let's make this stable sort. If the property is equal, we just compare voting power. If voting power is equal we compare moniker.